pendingTranslations.ftl

```diff
@@ -11,3 +11,27 @@ main-nav-link-faq-tooltip = Frequently asked questions
 
 footer-external-link-faq-label = FAQs
 footer-external-link-faq-tooltip = Frequently asked questions
+
+# Status Pill
+
+status-pill-action-needed = Action Needed
+status-pill-progress = In Progress
+status-pill-fixed = Fixed
+
+# Exposure Card
+
+exposure-card-family-members = Family members
+exposure-card-email = Type
+exposure-card-phone-number = Phone number
+exposure-card-address = Address
+# Variables:
+# $exposure_type is the number of exposures found for a particular type, e.g. 3 found 
+exposure-card-num-found = { $exposure_num } found
+exposure-card-description-info-for-sale-part-one = This site is selling and publishing <data_broker_link>details about you</data_broker_link>
+exposure-card-description-info-for-sale-part-two = Remove this profile to protect your privacy.
+# Variables:
+# $data_breach_company is the company assosciated with the data breach.
+# $data_breach_date is the date of the data breach.
+exposure-card-description-data-breach-part-one = Your information was exposed in the <data_breach_link>{ $data_breach_company } data breach on { $data_breach_date }</data_breach_link>
+exposure-card-description-data-breach-part-two = We'll walk you through the steps to fix it.
```

Comment:
nit: use smart quote for `We'll`

Not sure if we want to modify /docs/CODEOWNERS to include @flodolo as a codeowner for /pendingTranslations.ftl too. I'll let Flod decide in case it's too much noise to review pending vs waiting for them to land in locales/en/*.ftl.

 https://github.com/mozilla/blurts-server/blob/75e7d70988ddc122e2882feaa69aa807d0218819/docs/CODEOWNERS#L1-L2

Reply:
The linter should be also extended to check for these files
https://github.com/mozilla/blurts-server/blob/main/.github/workflows/reference_linter.yaml